src/main/resources/roleconfig/role_common.json

```diff
@@ -2001,6 +2001,36 @@
       }
     ]
   },
+  {
+    "name": "caseworker-privatelaw-externaluser-viewonly",
```

Comment:
Can we add unit tests to `src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java` as we do for other `OTHER_GOV_DEPT` roles such as [dwp, hmrc](https://github.com/hmcts/am-role-assignment-service/blob/50abff261fc17eabba78973dbb3b8daa4dd98320/src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java#L424-L425).

Reply:
Also add `listed-hearing-viewer` for PRIVATELAW to unit tests.

Reply:
Resolved in 37e10b1b96b9927511c31f2457353c826ca1c756